package.json

```diff
@@ -43,6 +43,7 @@
     "@reach/dialog": "0.2.9",
     "@styled-system/prop-types": "5.0.5",
     "@styled-system/theme-get": "5.0.5",
+    "@types/styled-system": "5.1.0",
```

Comment:
I have a theory that adding `@types/styled-system` as a dependency is all we have to do to fix the TypeScript errors people have been running into. I don't think we need to import types directly from `@types/styled-system`